src/main/java/io/ddavison/conductor/Locomotive.java

```diff
@@ -96,7 +110,17 @@ public Locomotive() {
 
         configuration = new LocomotiveConfig(testConfiguration, props);
 
-        Capabilities capabilities;
+        DesiredCapabilities capabilities;
+        
+        Capabilities extraCapabilities;
+		try {
+			extraCapabilities = configuration.capabilities().newInstance();
+		} catch (InstantiationException | IllegalAccessException e) {
+			e.printStackTrace();
```

Comment:
There should be logger used here instead printStackTrace. By using logger it will bea easier to add monitoring tools like Kibana.